components/brave_wallet/browser/internal/orchard_storage/orchard_storage.h

```diff
@@ -29,10 +29,10 @@ namespace brave_wallet {
 
 template <size_t const T>
 base::expected<std::optional<std::array<uint8_t, T>>, std::string>
-ReadSizedBlob(sql::Statement& statement, size_t position) {
-  auto columns = statement.ColumnCount();
+ReadSizedBlob(sql::Statement& statement, int position) {
```

Comment:
This method should go to .cc file close to `ReadUint32`, `ReadAmount`, etc.. And they also should get `position` parameter arg `int`

Reply:
I meant `ReadUint32`, `ReadAmount`,etc should also have `int` position

Reply:
Ah, gotcha. Thanks. Fixed.